ccm_web/client/src/components/Help.tsx

```diff
@@ -4,9 +4,6 @@ import { makeStyles, Typography, Link } from '@material-ui/core'
 const useStyles = makeStyles((theme) => ({
   helpText: {
     float: 'right'
-  },
-  helpLink: {
-    textDecoration: 'underline'
```

Comment:
I remember last time what I was working on Help, Janel suggested to add a link to Help as well. Now there is no need so removing it.